### PR TITLE
XREF corneal reflex

### LINKED ISSUE #8: add xref for  NBO:0000415 ! corneal reflex
```
GO:0060082 ! eye blink reflex

or demote:

synonym: "blink reflex" EXACT []
```

Original issue reported on code.google.com by `cmung...@gmail.com` on 14 Dec 2011 at 8:35

### PR DESCRIPTION
**Pull request by DitchingIt**
Fixes #8 xref to GO:0060082

### CODE REVIEW

File: src/ontology/nbo-edit.owl [hunk 1]
```diff
@@ -5113,6 +5113,7 @@ AnnotationAssertion(oboInOwl:created_by obo:NBO_0000415 "George Gkoutos")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:NBO_0000415 "blink reflex")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:NBO_0000415 "blinking")
 AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0000415 "behavior_ontology")
+AnnotationAssertion(Annotation(dc:contributor "0000-0002-4277-2151") oboInOwl:xref obo:NBO_0000415 "GO:0060082")
```
Comment (matentzn) [inline src/ontology/nbo-edit.owl:5116]:
```suggestion
AnnotationAssertion(Annotation(dc:contributor "http://orcid.org/0000-0002-4277-2151") oboInOwl:hasDbXref obo:NBO_0000415 "GO:0060082")
```

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/ontology/nbo-edit.owl (hunk 1 above)
Comment (matentzn) [reply src/ontology/nbo-edit.owl:5116]: Careful with the way you add xrefs an ORCIDs!

**Review by DitchingIt — COMMENTED** [review]
maybe confused by what is happening here.

File: src/ontology/nbo-edit.owl (hunk 1 above)
Comment (DitchingIt) [reply src/ontology/nbo-edit.owl]: I didn't know either was wrong. Thanks. I pressed the 'commit suggestion' button but now things seem to have stalled. Did it do anything? Do I have to go back to my original branch and make a change there myself?

Comment (DitchingIt) [reply src/ontology/nbo-edit.owl]: Not sure what is wanted from me. I will 'resolve conversation' in case that is what is needed???

**Review by matentzn — APPROVED** [review]
Nice.

**Pull request merged**